lib/matplotlib/axes/_axes.py

```diff
@@ -6132,6 +6135,10 @@ def hist(self, x, bins=None, range=None, normed=False, weights=None,
         if histtype == 'barstacked' and not stacked:
             stacked = True
 
+        if density is not None and normed is not None and normed != density:
+            raise ValueError("kwargs density and normed cannot have different\
```

Comment:
Minor style comment. Can you do:

```python
raise ValueError("kwargs density and normed cannot have different "
                 "values. Use density only, since normed will be deprecated.")